frontend/src/components/OurProjects.vue

```diff
@@ -1,60 +1,201 @@
 <template>
   <v-container fluid class="white-container">
+    <v-row>
+      <v-col>
+        <v-card flat color="transparent">
+          <v-card-text class="title">
+            <p class="blue-title">NASZE PROJEKTY</p>
+          </v-card-text>
+        </v-card>
+      </v-col>
+    </v-row>
     <v-layout row wrap>
       <v-flex
         id="items"
-        v-for="project in projects"
-        :key="project.name"
+        v-for="(project, index) in projects"
+        :key="index"
         xs12
         md6
         lg4
         xl3
       >
         <v-item-group>
-          <v-hover>
-            <v-card slot-scope="{ hover }">
-              <v-img :src="project.image" aspect-ratio="1.9"></v-img>
+          <v-hover v-slot="{ hover }">
+            <v-card @click.stop="clickImage(project)">
+              <v-img :src="project.imageAdress" aspect-ratio="1.9"></v-img>
+              <v-card-title id="card">
+                {{ project.name }}
+              </v-card-title>
               <v-expand-transition>
                 <div v-if="hover" id="card--reveal">
-                  <v-img id="hoverd-img" :src="hoveredImg"></v-img>
+                  <v-img id="card--hover" :src="hoveredImg"></v-img>
                 </div>
               </v-expand-transition>
             </v-card>
           </v-hover>
         </v-item-group>
       </v-flex>
     </v-layout>
+    <v-dialog v-model="dialog" width="50rem">
+      <app-modal-content :selectedProject="selectedProject"></app-modal-content>
+    </v-dialog>
   </v-container>
 </template>
 
 <script>
+import ModalContent from './ModalContent.vue';
+import projects_db from '../assets/projects';
```

Comment:
I would say that changing the name of what is imported is so rare situation, that I would say that we should be fine with just `projects` here.

Reply:
Yes it is. I wanted to differentiate both not to have var like projects: projects because I was afraid it won't work. It did so you were right.